src/instructlab/configuration.py

```diff
@@ -260,6 +260,7 @@ class _serve_vllm(BaseModel):
     """Class describing configuration of vllm serving backend."""
 
     llm_family: str = ""
+    max_startup_attempts: typing.Optional[int] = None
```

Comment:
@RobotSail mentioned in another review that pydantic might not always interpret typing.Optional properly so we might want something like `int | None = None` here

Reply:
Updated, thanks!